### PR TITLE
Annotated vlan_filter.c, xdp_filter.c, hash_func01.h

### PR DESCRIPTION
**Pull request by aashu2303**

### CODE REVIEW

File: projects/suricata-master/human_annotated/bypass_filter.c [hunk 1]
```diff
@@ -143,7 +143,7 @@ struct vlan_hdr {
...
+      "text": "/* Copyright (C) 2018 Open Information Security Foundation\n *\n * You can copy, redistribute or modify this Program under the terms of\n * the GNU General Public License version 2 as published by the Free\n * Software Foundation.\n *\n * This program is distributed in the hope that it will be useful,\n * but WITHOUT ANY WARRANTY; without even the implied warranty of\n * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n * GNU General Public License for more details.\n *\n * You should have received a copy of the GNU General Public License\n * version 2 along with this program; if not, write to the Free Software\n * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA\n * 02110-1301, USA.\n *"
```
Comment (sdsen) [inline projects/suricata-master/human_annotated/bypass_filter.c:146]: fix the typo by closing the comment.

File: projects/suricata-master/human_annotated/xdp_filter.c [hunk 2]
```diff
@@ -1557,7 +1557,7 @@ static int __always_inline filter_ipv6(struct xdp_md *ctx, void *data, __u64 nh_
   "call_depth": -1,
   "humanFuncDescription": [
     {
-      "description": "",
+      "description": "This hash function filter, filters the Ethernet packets based on the IP packet version. If GLOBAL_BYPASS flag is set, then the key0 is initialized to 0 and is checked in the global_bypass map. If the key exists, the same key0 is checked in tx_peer_int map. If the key doesn't exist, the packet is dropped. Othereise, the tx_port is written into the tx_peer map. If GLOBAL_BYPASS is not set, the fields vlan0 is set to the last 12 bits of the VLAN TCI field and vlan1 is set to 0 if the VLAN_TRACKING is off else it is set to the last 12 bits of the VLAN TCI field(since vlan0 is stripped by the OS). Based on the version of IP packets, the function filter_ipv4 or filter_ipv6 is invoked accordingly. If it is neither of them -1 is returned.",
       "author": "",
```
Comment (sdsen) [inline projects/suricata-master/human_annotated/xdp_filter.c:1561]: add author details. Also, please add personal email which you plan to keep using long term.

**Review by sdsen — CHANGES_REQUESTED** [review]

File: projects/suricata-master/human_annotated/xdp_filter.c (hunk 2 above)
Comment (aashu2303) [reply projects/suricata-master/human_annotated/xdp_filter.c]: Updated the email address and author details

File: projects/suricata-master/human_annotated/bypass_filter.c (hunk 1 above)
Comment (aashu2303) [reply projects/suricata-master/human_annotated/bypass_filter.c]: Fixed it

**Review by sdsen — APPROVED** [review]
lgtm

**Pull request merged**